letsencrypt/display/util.py

```diff
@@ -61,10 +75,11 @@ def menu(self, message, choices,
 
         :param str ok_label: label of the OK button
         :param str help_label: label of the help button
+        :param dict _kwargs: absorbs default / cli_args
 
-        :returns: tuple of the form (`code`, `tag`) where
```

Comment:
I'm not sure that they were intentionally removed, I think there were multiple variants of this method's docstring in the code, some with and some without backticks.  I'll try to standardise on including them.